WinUIGallery/Samples/ControlPages/Design/IconographyPage.xaml

```diff
@@ -184,92 +184,99 @@
-                        <TextBlock
-                            FontSize="12"
-                            Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                            Text="Code glyph" />
-                        <controls:SampleCodePresenter Code="{Binding SelectedItem.CodeGlyph, Mode=OneWay}" Style="{StaticResource CodeValuePresenterStyle}" />
-                        <TextBlock
-                            FontSize="12"
-                            Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                            Text="XAML" />
-                        <controls:SampleCodePresenter x:Name="XAMLCodePresenter" Style="{StaticResource CodeValuePresenterStyle}" />
-                        <TextBlock
-                            FontSize="12"
-                            Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                            Text="C#" />
-                        <controls:SampleCodePresenter x:Name="CSharpCodePresenter" Style="{StaticResource CodeValuePresenterStyle}" />
-                        <TextBlock
-                            Margin="0,4,0,0"
-                            FontSize="12"
-                            Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                            Text="Tags" />
-                        <ItemsView
-                            x:Name="TagsItemsView"
-                            Margin="0,8,0,4"
-                            x:Load="{x:Bind SelectedItem.Tags, Mode=OneWay, Converter={StaticResource emptyCollectionToFalseConverter}}"
-                            IsItemInvokedEnabled="True"
-                            ItemInvoked="TagsItemsView_ItemInvoked"
-                            ItemsSource="{x:Bind SelectedItem.Tags, Mode=OneWay}"
-                            SelectionMode="None">
-                            <ItemsView.Layout>
-                                <toolkit:WrapLayout HorizontalSpacing="4" VerticalSpacing="4" />
-                            </ItemsView.Layout>
-                            <ItemsView.ItemTemplate>
-                                <DataTemplate x:DataType="x:String">
-                                    <ItemContainer
-                                        HorizontalAlignment="Left"
-                                        AutomationProperties.Name="{x:Bind}"
-                                        CornerRadius="12">
-                                        <Grid
-                                            Padding="8,3,8,4"
-                                            Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
-                                            BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
-                                            BorderThickness="1"
+                    <ScrollViewer Padding="16,16,8,16">
```

Comment:
In general; setting a **Padding** on a **Scrollviewer** may produce "ugly" results at the borders. Setting the margin on the first content will solve that.
![image](https://github.com/user-attachments/assets/a6e75ce3-8538-453e-9f04-ead15cb1126b)

Reply:
Great catch! Resolved!